test/arm/neon/ld1_dup.c

```diff
@@ -1258,6 +1258,303 @@ test_simde_vld1q_dup_u64 (SIMDE_MUNIT_TEST_ARGS) {
 #endif
 }
 
+static int
+test_simde_vld1_dup_p8 (SIMDE_MUNIT_TEST_ARGS) {
+  static const struct {
+    simde_poly8_t a;
+    simde_poly8_t unused;
+    simde_poly8_t r[8];
```

Comment:
I wrote test cases based on the original ones. I think the reason for adding one unused element is to verify the correctness of the implementation. This is because the elements in the structure are stored in memory in sequential order, meaning that the variables in memory are arranged as follows:
`a`, `unused`, `r[0]`, `r[1]`, ... , `r[7]`.
Therefore, the extra slot is added to double-check the function's behavior for accuracy.